test/graphql/queries/profile_query_test.rb

```diff
@@ -40,7 +40,8 @@ class ProfileQueryTest < ActiveSupport::TestCase
       }
     GRAPHQL
 
-    profiles(:one).update account: accounts(:test)
+    profiles(:one).update account: accounts(:test),
+                          parent_profile: profiles(:two)
```

Comment:
Canonical (`parent_profile == nil`) profiles are always visible by everyone based on our current Pundit policy. We need to write a data migration to set parent_profile, matched via benchmark and ref_id, on existing profiles that are associated with an account, or this Pundit policy will give unauthorized access.

Reply:
I was going to merge this one to CI, but if I do, it looks like the policy will block the "create policy" workflow, as there will not be any policies with `parent_profile_id` set for any user. It looks like we need the migration either on this PR or some PR that is merged before this one in order to be able to merge without blocking that workflow.